lib/spender/planning/log_category.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Spender.Planning.LogCategory do
+  @moduledoc """
+  This module holds changeset functions used to work with the LogCategory
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Spender.{
+    MoneyLogs.Budget
+    }
```

Comment:
Could you please consider indenting this correctly?

Reply:
Okay